classes/plugins/ThemePlugin.inc.php

```diff
@@ -17,6 +17,29 @@
 
 abstract class ThemePlugin extends LazyLoadPlugin {
 	/**
+	 * Collection of styles
+	 *
+	 * @see self::_registerStyles
+	 * @param array $styles
```

Comment:
We're using...

    @param $variableName VariableType Description goes here...